terraform/azure/main.tf

```diff
@@ -41,3 +55,105 @@ resource "azurerm_storage_account" "orsoarpadev" {
     environment = "dev"
   }
 }
+
+resource "azurerm_app_service_plan" "orsoarpadev" {
+  name                = module.app_service_plan_name.result
+  location            = azurerm_resource_group.orsoarpadev.location
+  resource_group_name = azurerm_resource_group.orsoarpadev.name
+
+  sku {
+    tier = "Free"
+    size = "F1"
+  }
+
+  tags = {
+    environment = "dev"
+  }
+}
+
+resource "azurerm_app_service" "orsoarpadev" {
+  name                = module.app_service_name.result
+  location            = azurerm_resource_group.orsoarpadev.location
+  resource_group_name = azurerm_resource_group.orsoarpadev.name
+  app_service_plan_id = azurerm_app_service_plan.orsoarpadev.id
+  https_only = true
+
+  site_config {
+    dotnet_framework_version = "v5.0"
+    scm_type                 = "GitHub"
+    http2_enabled = true
+    health_check_path = "/health"
+  }
+
+  app_settings = {
```

Comment:
these settings should be passed from outside

Reply:
then define a variable with type "map" and define the value as key:value pairs.

Reply:
If you are using a map variable to get the input from outside, then you may have to extend this map here with 

_**CorsConfiguration:AllowedOrigins:0**_, as the value for it depends on the **_azurerm_storage_account_** resource like I pointed below.

Reply:
pls check the variables I created

Reply:
LGTM.